po/de.po

```diff
@@ -2901,13 +2901,13 @@ msgstr "Beste Ziele: Fahrzeuge"
 #: data/mp/messages/strings/resstrings.txt:354
 #: po/custom/fromJson.txt:1125
 msgid "Replaces Medium Cannon"
-msgstr "Ersetzt mittleres Geschütz"
+msgstr "Ersetzt mittleres Kanone"
```

Comment:
Here you forgot to replace the word "mittleres" with "mittlere".

Perhaps that word should also be capitalized as part of a proper noun.
This would imply that any other weapon names should follow the same convention.

Reply:
Done. I capitalized "Leichte Kanone", Mittlere Kanone", "Schwere Kanone", "Schweres Maschinengewehr" and "Schwerer Laser"